app/views/chat_rooms/show.html.erb

```diff
@@ -1,11 +1,21 @@
 <div class="container">
     <h1><%= @chat_room.title %></h1>
-    <div class="chat-box">
+    <div id="chat-box">
         
         <%= render @chat_room.messages %>
     
     </div>
     <%= link_to "New message", new_chat_room_message_path(@chat_room), class:"btn btn-success" %>
     
     
-</div>
\ No newline at end of file
+</div>
+
+
+<script>
+window.onload = function(){
+    var scrollBox = document.querySelector("#chat-box");
```

Comment:
You can also use the JQuery shorthand for this: `var scrollbox = $('#chat-box')`